lfucg-exactions/server/accounts/viewsets.py

```diff
@@ -193,7 +193,7 @@ class ProjectCostEstimateViewSet(viewsets.ModelViewSet):
 
 
     def get_queryset(self):
-        queryset = ProjectCostEstimate.objects.all()
+        queryset = ProjectCostEstimate.objects.all().exclude(is_active=False)
```

Comment:
no need to have all() before exclude(). Same comment for the rest of the same changes